ehitchhiking-ui/src/app/shared/constants/modal-constants.ts

```diff
@@ -5,3 +5,4 @@ export const MAT_DIALOG_WIDTH_SM = '23rem';
 export const MAT_DIALOG_WIDTH_MD = '40rem';
 export const MAT_DIALOG_WIDTH_LG = '55rem';
 export const DELETE_ROUTE_MARKER = -1;
+export const TRIPS_IN_LIST = 4;
```

Comment:
bad naming: TRIPS_IN_LIST. Pls rename to NUMBER_OF_TRIPS_VISIBLE_ON_PAGE

Reply:
renamed